src/main/kotlin/build/buf/intellij/annotator/BufAnalyzePass.kt

```diff
@@ -103,7 +104,7 @@ class BufAnalyzePass(
         if (file !is PbFile) return
 
         if (annotationInfo == null) {
-            disposable = myProject
+            disposable = appService
```

Comment:
Missed this from previous refactoring - project shouldn't be used as a disposable.